partcad-ide-vscode/resources/js/inspector.js

```
@@ -0,0 +1,408 @@
+
+
+  // Handle messages sent from the extension to the webview
+  window.addEventListener('message', event => {
+    const message = event.data; // The json data that the extension sent
+    let config = {};
+    if (message.type === 'package' || message.type === 'sketch' || message.type === 'interface' || message.type === 'part' || message.type === 'assembly') {
+      config = message.obj['config'];
+    }
+    switch (message.type) {
+      case 'clear':
+        let contents = document.querySelector('.contents');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)